README.md

```diff
@@ -13,7 +13,7 @@ This field plug-in is designed to help execute the Early Grade Reading Assessmen
 
 For details on these tests, please consult the [USAID EGRA Toolkit](https://pdf.usaid.gov/pdf_docs/PA00M4TN.pdf).
 
-[![Download now](extras/beta-release-download.jpg)](https://github.com/surveycto/egra-test/raw/master/egra-test.fieldplugin.zip)
+[![Download now](extras/beta-release-download.jpg)](https://github.com/surveycto/timed-grid-test/raw/develop/timed-grid-test.fieldplugin.zip)
```

Comment:
@mofya1 this link is pointing to the develop branch. you should change /raw/**develop**/... to /raw/**master**/... in all these links. 

Make sure to check the links to the other things like sample forms as well. The part of the URL immediately after /raw/ is the branch name, which should be "master" for all of these links.

Reply:
Ah yes! Made updates to the other links too. Thanks.